devel/iniparser/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        ndevilla iniparser 4.1 v
+github.tarball_from archive
+revision            0
+
+categories          devel
+platforms           darwin
+maintainers         {@frankdean fdsd.co.uk:frank.dean} openmaintainer
+
+license             MIT
+
+description         An ini file parser library for C
+
+long_description    ${description}
+
+patchfiles          Makefile.diff
+
+checksums           rmd160  06b85b087760fc40ab5a2568db663df25b13ccb6 \
+                    sha256  960daa800dd31d70ba1bacf3ea2d22e8ddfc2906534bf328319495966443f3ae \
+                    size    44732
+
+use_configure       no
+
+set libdir          ${prefix}/lib
+set includedir      ${prefix}/include/iniparser
+
+build.target        {}
```

Comment:
leaving it empty should be sufficient and is typically done, so just remove the `{}`

Reply:
Removed the braces.